posts/2020-05-15-six-years-of-rust.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Six Years of Rust"
+author: The Rust Core Team
+---
+
+Today marks [Rust]'s sixth birthday since it went 1.0 in 2015. A lot has changed since then and especially over the past year, and Rust was no different. In 2020, there was no foundation yet, no const generics, and a lot organisations were still wondering whether Rust was production ready. 
+
+In the midst of the COVID-19 pandemic, hundreds of Rust's global distributed set of team members and volunteers shipped over 9 major releases of Rust and multiple patch releases. Today, "Rust in production" isn't a question, but a statement. The newly founded Rust foundation's has several members who value using Rust in production enough to help continue to support and contribute to its open development ecosystem.
```

Comment:
nit: we shipped minor releases in the semver sense - probably makes sense to find a non-semver-used word here.

Reply:
Yeah, I did notice that and wasn't really happy with it, but couldn't come up with a better alternative. Happy to hear  alternatives though.

Reply:
Perhaps "nine new stable releases of Rust, in addition to various bugfix releases"?